### PR TITLE
Support external documents and photos

### PR DESCRIPTION
**Pull request by aykxt**

### CODE REVIEW

File: lib/grammers-client/src/types/input_message.rs [hunk 1]
```diff
@@ -134,6 +134,23 @@ impl InputMessage {
         self
     }
 
+    /// Include an external photo in the message.
+    ///
+    /// The server will download and compress the image and convert it to JPEG format if
+    /// necessary.
```
Comment (Lonami) [inline lib/grammers-client/src/types/input_message.rs:140]:
```suggestion
    /// The Telegram server will download and compress the image and convert it to JPEG format if necessary.
```

File: lib/grammers-client/src/types/input_message.rs [hunk 2]
```diff
@@ -158,6 +175,22 @@ impl InputMessage {
         self
     }
 
+    /// Include an external file as a document in the message.
+    ///
+    /// You can use this to send videos, stickers, audios, or uncompressed photos.
+    ///
```
Comment (Lonami) [inline lib/grammers-client/src/types/input_message.rs:181]:
```suggestion
    ///
    /// The Telegram server will download and compress the image and convert it to JPEG format if necessary.
    ///
    ///
```

File: lib/grammers-client/src/types/input_message.rs (hunk 1 above)
Comment (aykxt) [reply lib/grammers-client/src/types/input_message.rs:140]: I copied the text from the `.photo()` method. Should I add the "Telegram" there too?

Comment (Lonami) [reply lib/grammers-client/src/types/input_message.rs:140]: Oh, didn't even realize. Sure.

**Pull request merged**